py/torch_tensorrt/dynamo/runtime/_PythonTorchTensorRTModule.py

```diff
@@ -52,6 +53,7 @@ def __init__(
         self.profiling_enabled = (
             profiling_enabled if profiling_enabled is not None else False
         )
+        self.settings = settings
```

Comment:
Where do you use the `self.settings`?

Reply:
https://github.com/pytorch/TensorRT/pull/2886/files#:~:text=if%20settings%20is,.DEBUG)